src/en/space-station-14/round-flow/antagonists/Xenoborgs.md

```diff
@@ -0,0 +1,194 @@
+
+As borgs with specific laws and being instantly valid they are very heavy discouraged from helping the crew or not being a antag.
+
+They are instantly visible as a strange cyborg, examining them will make it clear that they are not friendly and if you don't figure out they mean harm, maybe that cyborg shooting lasers at you will help you figure it out.
+
+As they are essentially cyborgs with special laws and modules, any know how of normal cyborgs transfers, both to players playing as xenoborgs but as crew fighting the xenoborgs. The only things that might need specific instruction are the unique radios they use. The mothership might also need some specific instruction given how weird their role is as a sentient lathe with AI like vision
+
+If you lack equipment or they get you via surprise attack and you are alone, is very likely you will die with near to no option on counter play. but in that case, is most certainly you will be able to play as a xenoborg and still have fun and enjoy the round. If you are aware of xenoborgs in the station and are well prepared you can have meaningful fights where the result is not certain.
+
+<!--
+Consider addressing:
+- How does the feature align with our [Core Design Principles](../space-station-14/core-design/design-principles.md) and game philosphy?
+- What makes this feature enjoyable or rewarding for players?
+- Does it introduce meaningful choices, risk vs. reward, or new strategies?
+- How does it enhance player cooperation, competition, or emergent gameplay?
+- If the feature is a new antagonist, how does it fit into the corresponding [design pillars](../space-station-14/round-flow/antagonists.md)?
+-->
+
+## Roundflow & Player interaction
+
+It should be a rare roundstart subgame mode. round start to give them the proper time to grow and become a issue, and if they grow they will affect the round. given that they might lose at the start it should be a subgame mode and accompanied by other antags that they can have fun interactions with.
+
+I wish players playing xenoborg try to be strategic and stealthy in the beginning so they can actually grow and become a threat to the station. This is enforced by the fact that they start with low numbers and are not very strong alone and they are instantly valid and their future threat can encourage the crew to arm if they manage to get discovered.
+
+It will most likely start interacting with cargo, as salvagers and cargo techs are the best starting targets. As they grow and get more people they will be discovered and will have more contact with security but will affect medbay, since they will be healing anyone that is injured but not taken away by borgs, they can affect science as they will try to make weapons and flashes to deal with the borgs, and they will have many MMIs with brains to borg if they destroy xenoborgs. the xenoborgs might also decide to sabotage the station affecting the engineering department.
```

Comment:
```suggestion
It will most likely start interacting with cargo, as salvagers and cargo techs are the best starting targets. As they grow and get more people they will be discovered and will have more contact with security but will affect medbay, since they will be healing anyone that is injured but not taken away by borgs, they can affect science as they will try to make weapons and flashes to deal with the borgs, and they will have many MMIs with brains to borg if they destroy xenoborgs. The xenoborgs might also decide to sabotage the station affecting the engineering department.
```